src/Uno.UI.Runtime.Skia.WebAssembly.Browser/ts/Runtime/BrowserRenderer.ts

```diff
@@ -36,13 +36,15 @@ namespace Uno.UI.Runtime.Skia {
 			var w = width * scale
 			var h = height * scale;
 
-			if (this.canvas.width !== w)
-				this.canvas.width = w;
-			if (this.canvas.height !== h)
-				this.canvas.height = h;
-
-			this.canvas.style.width = `${width}px`;
-			this.canvas.style.height = `${height}px`;
+			if (this.canvas) {
```

Comment:
Should we include the rest of the method body within this validation? That way, we avoid performing operations and discard them because `this.canvas` is not ready yet.